docs/python/editing.md

```diff
@@ -64,12 +64,13 @@ You can also customize the general behavior of autocomplete and IntelliSense, ev
 
 For autocomplete and IntelliSense issues, check the following causes:
```

Comment:
Should we change the way we are describing this table to broaden the scope? May not be necessary  "For common Intellisense and Python editing issues, check the following causes:"?

Reply:
I like that!

Reply:
Yp duped spruce.. love it